internal/pkg/levee/testdata/src/example.com/tests/panic/tests.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package panic
+
+import (
+	"example.com/core"
+)
+
+func TestPanicIsASink(source core.Source) {
+	panic(source) // want "a source has reached a sink"
+}
```

Comment:
I see in the [`ssa.Panic` documentation](https://pkg.go.dev/golang.org/x/tools/go/ssa#Panic) that `go panic` and `defer panic` are handled differently.

We should decide if those are out of scope at present, but might want to introduce (potentially suppressed) tests and open an issue for it.

Reply:
Good idea, Ill keep that out of scope of this for now. Ill add some tests with todos for those and open an issue. I don't think `defer panic` and `go panic` are commonly used though.

Reply:
Done.